utils.ts

```diff
@@ -127,7 +119,9 @@ export const addTripleToTtl = function (
   o: string,
 ) {
   // eslint-disable-next-line prettier/prettier
-  return `${ttl} ${sparqlEscapeUri(s)} ${sparqlEscapeUri(p)} ${sparqlEscapeString(o)} .`;
+  return `${ttl} ${sparqlEscapeUri(s)} ${sparqlEscapeUri(
+    p,
+  )} ${sparqlEscapeString(o)} .`;
```

Comment:
Might need to add
// prettier-ignore
to stop prettier from modifying this line